### PR TITLE
GSoC 2018

### PR DESCRIPTION
**Pull request by tsegismont**
Updated the GSoC page and added a blog post

### CODE REVIEW

File: src/site/blog/posts/2018-03-13-googlesummerofcode-2018.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+This year, the Eclipse Vert.x project participates through the [Eclipse organization](https://summerofcode.withgoogle.com/organizations/6364590460370944/).
+Make sure to review our [GSoC 2018 ideas](https://wiki.eclipse.org/Google_Summer_of_Code_2018_Ideas#Eclipse_Vert.x) and to submit before March, 27!
+
+## Assessment application
+
+As we did before, we ask candidates to [implement a simple Vert.x application](https://github.com/tsegismont/vertx-gsoc-2018-qotd).
+This helps us make sure candidates have a basic understanding of asynchronous programming and the Vert.x toolkit.
+**But submit your proposal even if not done with the assessment application!**
+Google will not extend the submission period but we can continue reviewing assessments while evaluating the submitted proposals.
```
Comment (vietj) [inline src/site/blog/posts/2018-03-13-googlesummerofcode-2018.md]: I should you say that candidates can ask questions through the issue tracker of the application ?

File: src/site/blog/posts/2018-03-13-googlesummerofcode-2018.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Questions?
+
+If you have questions, feel free to ask possible mentors via email or on our [community channels](/community).
```
Comment (vietj) [inline src/site/blog/posts/2018-03-13-googlesummerofcode-2018.md]: people perhaps can ask questions via the repository issue tracker ?

Comment (tsegismont) [reply src/site/blog/posts/2018-03-13-googlesummerofcode-2018.md]: I didn't want to put too much info here. The issue tracker is for questions relative to the assessment app.

**Comment by tsegismont** [comment]
@vietj thanks for taking a look!

**Pull request merged**